maquetado/scss/tech.scss

```diff
@@ -58,3 +54,18 @@ $tech-logo-size: 100px;
   text-align: center;
   width: 130px;
 }
+
+@media only screen and (max-width: 1024px) {
+  .technology {
+    border-top: 1px solid $green;
+  }
+
+  .wrap {
+    display: inline;
+  }
+
+  .first-card {
+    border: none;
```

Comment:
just to learn, instead of using this, add a pseudo-element to technolgy, something like `nth-child` or `first-child`